packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -17,4 +17,20 @@ describe("<DateRangeInput>", () => {
         const component = mount(<DateRangeInput />);
         expect(component.find(InputGroup).length).to.equal(2);
     });
+
+    it("startInputProps.inputRef receives reference to HTML input element", () => {
+        const inputRef = sinon.spy();
+        // full DOM rendering here so the ref handler is invoked
+        mount(<DateRangeInput startInputProps={{ inputRef }} />);
+        expect(inputRef.calledOnce).to.be.true;
+        expect(inputRef.firstCall.args[0]).to.be.an.instanceOf(HTMLInputElement);
```

Comment:
Is the `calledOnce` part the useful piece here? i cant see these tests doing anything other than testing React and typescript.